hack/build/deploy_rhel.sh

```diff
@@ -2,10 +2,60 @@
 
```

Comment:
could we move this to a makefile step? as it is it contains various part of existing scripts, which could be better handled as make prerequisites.
That would also make it easier for us to keep it updated and break it less 😄

Reply:
See my above comments. If things continue on porting DTO to power/Z then I have no problem moving this into the Makefile. I fear the whole thing has reached a dead end.